cluster.go

```diff
@@ -331,13 +341,22 @@ func (c *Cluster) getTopo(p Client) (ClusterTopo, error) {
 // to new instances and removing ones from instances no longer in the cluster.
 // This will be called periodically automatically, but you can manually call it
 // at any time as well
-func (c *Cluster) Sync() error {
+func (c *Cluster) Sync(args ...bool) error {
```

Comment:
The method signature of `Sync` cannot be changed, and I think calls to `Sync` should not be subject to the silence flag at all. You will need to change the invocation of `Sync` in `NewCluster` to use `sync` instead.